package-overrides/github/chieffancypants/angular-hotkeys@1.4.5.json

```diff
@@ -0,0 +1,15 @@
+{
+  "directories": {
+    "lib": "build"
```

Comment:
We're not encouraging this anymore for exactly the reason that it can break things for others as you experienced.